pkg/js/helpers.js

```diff
@@ -410,6 +410,9 @@ var NAPTR = recordBuilder('NAPTR', {
     },
 });
 
+var NETLIFY = recordBuilder('NETLIFY')
+var NETLIFYv6 = recordBuilder('NETLIFYv6')
```

Comment:
Are NETLIFY and NETLIFYv6 needed if we're going to ignore them?

Reply:
Resolved in dc10565c